tests/flow/tests_pytorch.py

```diff
@@ -652,15 +653,15 @@ def test_pytorch_modelscan_scriptscan(env):
 
     model_pb = load_file_content('pt-minimal.pt')
 
-    ret = con.execute_command('AI.MODELSTORE', 'm1', 'TORCH', DEVICE, 'TAG', 'm:v1', 'BLOB', model_pb)
+    ret = con.execute_command('AI.MODELSTORE', 'm1', 'TORCH', DEVICE, 'TAG', 'm:v2', 'BLOB', model_pb)
```

Comment:
explain this tag?

Reply:
by mistake, this has changed as a result of refactoring the name of the dir "v1" to "v2" (not important...)